lib/web/servers.go

```diff
@@ -143,17 +143,32 @@ func (h *Handler) clusterDatabaseGet(w http.ResponseWriter, r *http.Request, p h
 		return nil, trace.Wrap(err)
 	}
 
-	dbServer, err := fetchDatabaseServerByDatabaseName(r.Context(), clt, r, databaseName)
+	dbServers, err := fetchDatabaseServersWithName(r.Context(), clt, r, databaseName)
```

Comment:
nit: any benefit using `fetchDatabaseServersWithName` versus using the unified resource api here? maybe we should switch to the unified resource API on Web UI side.